### ISSUE TITLE
[Bug] Drive: unable to download files that ask for location before download (large files)

### ISSUE THREAD

**Issue by TimmyBoi155** [vladimiry/ElectronMail#677]
In vanilla Chromium, I have noticed that files anywhere around 0.5/0.6GB+ will first ask for the file location, then download the file. But files where it is something like 100MB or less, it will download first, then ask you where to put the file. Again, in vanilla Chromium, all of this works fine, and this is the general behavior right now.

Now, in ElectronMail, If i try to do these larger type files, it will not ask for the file location first, but instead just start downloading, until it shows Success/"Downloaded" in green color, like it's complete and successful. And then it will not ask for the file location, either before or after the download....  However, the smaller files like 100MB or something, it will download first, then ask for file location to save, and will work fine

So basically any larger files around 500MB+ (i dont know the exact threshold, tested 600mb and 1gb and 6gb), will not work properly in ElectronMail.

I'm using the Flatpak version on XFCE. If you want me to do any other specific testing, just let me know.

**Comment by TimmyBoi155** [comment]
Vladimir, thanks for responding.

I did a bit of testing

In flatseal, I gave the app permission for all system files (filesystem=host), like so:
![image](https://github.com/vladimiry/ElectronMail/assets/88632743/24d2ce73-171a-4427-a605-3e06523de278)
After which, I restarted ElectronMail and tried another large file again, but still did not work.

Also, I decided to clear out the log file, start up the app fresh and go straight to drive, and try to download a large file (didnt work of course), then switch to email to see that it logged me out, then logged in again, then close the app. Then I uploaded the log file here, you can see it at this pastebin

https://pastebin.com/raw/keL1epH1

By the way, this message:
`Invocation timeout of calling "notification" method on "electron-mail:webview-api:primary" channel with 25000ms timeout`
happens as soon as I login to my account with a red banner at the top of the window.

Let me know if there is anything else I can do :) Or if there is anything I can help test.

**Comment by TimmyBoi155** [comment]
Vladimir, just out of curiosity, I tried out the same problematic large files on the AppImage, and it also did not work, so this seems to suggest that Flatpak is not the issue. Maybe you need to update the app to the newest Proton code?

**Comment by jarrard** [comment]
got a similar issue but only just started happening where I try and DL attachments and it locks EM up.
